pkgs/ffigen/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 11.0.0
```

Comment:
Maybe make it `-wip` for now, then we can add more breaking changes before releasing.

I'd like at least to also have @liamappelbe add some similar functionality for errors from ObjC.

Reply:
Sure